### DISCUSSION TITLE
ImageManager class is missing

### DISCUSSION THREAD

**Discussion by praction1988** [uvdesk/community-skeleton#770] [General]
I am deploying UVDESK on ubantu 22.04

but seems installing uvdesk community version is next to impsiable

Class "Intervention\Image\ImageManager" not found while loading "App\Controller\ImageCache\ImageManager".
![Screenshot 2024-08-26 at 12 11 42 AM](https://github.com/user-attachments/assets/765599bc-6cb7-407a-9a70-5a16cf73e142)

**Comment by papnoisanjeev** [comment]
@praction1988 
we have released a new version v1.1.7 please check now.
